snuba/clusters/storage_sets.py

```diff
@@ -0,0 +1,33 @@
+from typing import Mapping
+
+from enum import Enum
+from snuba.datasets.storages import StorageKey
+
+
+class StorageSetKey(Enum):
```

Comment:
Now this is the only definition of what a storage set is. It needs a docstring ecplaining what a storage set it.